platform/uboot/uboot.mk

```diff
@@ -22,6 +22,9 @@ UBOOT_OUT := $(PRODUCT_OUT)/obj/UBOOT_OBJ
 
 SYSFS_MMC0_PATH ?= soc/1c0f000.mmc
 SYSFS_MMC1_PATH ?= soc/1c11000.mmc
+RKTRUST_INI := RK3399TRUST.ini
```

Comment:
Consider putting this under "ifdef rockchip", and use full path , like: $(ROCKCHIP_FIRMWARE_DIR)/RKTRUST/RK3399TRUST.ini
and use `nodir` macro to get name only.

Reply:
It would be even better to move the `RKTRUST/RK3399TRUST.ini` into opi4.mk,